backend/whisper_with_diarization_as_methods.py

```diff
@@ -21,11 +21,13 @@ def validate_audio_file(audio_file_path: str) -> bool:
         - Audio file inputs are either .wav or .mp3. Whisper can process more audio file inputs, but the checking for
         other "types" of files has not been implemented yet
     """
-    validate_audio_path_msg = magic.from_file(audio_file_path)
-    if "WAVE audio" in validate_audio_path_msg or "Audio file" in validate_audio_path_msg: # TODO: 1st cond => .wav file. 2nd cond => .mp3 files
-        return True
-    else:
-        return False
+    validate_audio_path_msg = magic.from_file(audio_file_path, mime=True)
+    supported_file_extensions = {"mpeg", "mp4", "wav", "webm", "flac", "ogg"}
```

Comment:
On my computer, it considers mp4 as an invalid file format while m4a is considered a valid file format. Also something else I wanted to mention was that mp4 files don't play on my computer since I only have QuickTime Player to play audio files. So I think that if the user's computer doesn't have an application that can open a particular file they are trying to run the tool on, the tool will consider the file as invalid.

Reply:
Hm that is strange, I'll look into this more

Reply:
@carmen-chau It was the Spanish .mp4 file that was sent in the whisper app slack channel